kdeinit5: preparing to launch '/usr/lib/kf5/klauncher'
kdeinit5: Launched KLauncher, pid = 517, result = 0
kdeinit5: opened connection to :0
kdeinit5: preparing to launch 'libkdeinit5_kded5'
Could not open kded5 using a library: Cannot load library libkdeinit5_kded5: (libkdeinit5_kded5: cannot open shared object file: No such file or directory)
kdeinit5: Launched KDED, pid = 533 result = 0
kdeinit5: preparing to launch 'libkdeinit5_kcminit_startup'
kdeinit5: Launched 'kcminit_startup', pid = 534 result = 0
kdeinit5: Got EXEC_NEW '/usr/bin/kaccess' from launcher.
kdeinit5: preparing to launch '/usr/bin/kaccess'
kdeinit5: Got SETENV 'GTK_RC_FILES=/etc/gtk/gtkrc:/root/.gtkrc:/root/.config/gtkrc' from launcher.
kdeinit5: Got SETENV 'GTK2_RC_FILES=/etc/gtk-2.0/gtkrc:/root/.gtkrc-2.0:/root/.config/gtkrc-2.0' from launcher.
kdeinit5: Got SETENV 'XCURSOR_THEME=Oxygen_White' from launcher.
kdeinit5: PID 534 terminated.
kdeinit5: Got EXEC_NEW '/usr/lib/kf5/kconf_update' from launcher.
kdeinit5: preparing to launch '/usr/lib/kf5/kconf_update'
kdeinit5: PID 586 terminated.
kdeinit5: Got SETENV 'SESSION_MANAGER=local/aaa:@/tmp/.ICE-unix/574,unix/aaa:/tmp/.ICE-unix/574' from launcher.
kdeinit5: Got EXEC_NEW '/usr/lib/DiscoverNotifier' from launcher.
kdeinit5: preparing to launch '/usr/lib/DiscoverNotifier'
Connection failure: Connection refused
pa_context_connect() failed: Connection refused
blueman-applet 15.53.39 ERROR    PluginManager:90 load_plugin: Unable to load plugin module AppIndicator
Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/blueman/plugins/applet/AppIndicator.py", line 7, in <module>
    require_version('AppIndicator3', '0.1')
  File "/usr/lib/python3.8/site-packages/gi/__init__.py", line 129, in require_version
    raise ValueError('Namespace %s not available' % namespace)
ValueError: Namespace AppIndicator3 not available

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/blueman/main/PluginManager.py", line 88, in load_plugin
    importlib.import_module(self.module_path.__name__ + ".%s" % plugin)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/usr/lib/python3.8/site-packages/blueman/plugins/applet/AppIndicator.py", line 9, in <module>
    raise ImportError("AppIndicator3 not found")
ImportError: AppIndicator3 not found
blueman-applet 15.53.39 WARNING  TransferService:189 _make_share_path: Failed to get Download dir from XDG
blueman-applet 15.53.39 WARNING  PluginManager:147 __load_plugin: Not loading DhcpClient because its conflict has higher priority
blueman-applet 15.53.39 WARNING  PluginManager:147 __load_plugin: Not loading PPPSupport because its conflict has higher priority
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/desktop.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/desktop.so'
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/file.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/file.so'
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/file.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/file.so'
blueman-applet 15.54.04 WARNING  DiscvManager:109 update_menuitems: warning: Adapter is None
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/tags.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/tags.so'
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/trash.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/trash.so'
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/file.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/file.so'
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/file.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/file.so'
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/file.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/file.so'
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/thumbnail.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/thumbnail.so'
kdeinit5: PID 833 terminated.
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/file.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/file.so'
kdeinit5: PID 832 terminated.
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/file.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/file.so'
kdeinit5: PID 863 terminated.
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/tags.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/tags.so'
kdeinit5: PID 765 terminated.
kdeinit5: PID 774 terminated.
kdeinit5: PID 828 terminated.
kdeinit5: PID 829 terminated.
kdeinit5: PID 830 terminated.
kdeinit5: PID 834 terminated.
kdeinit5: PID 882 terminated.
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/tags.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/tags.so'
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/trash.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/trash.so'
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/file.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/file.so'
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/file.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/file.so'
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/thumbnail.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/thumbnail.so'
kdeinit5: PID 929 terminated.
kdeinit5: PID 767 terminated.
kdeinit5: PID 932 terminated.
kdeinit5: PID 928 terminated.
kdeinit5: PID 930 terminated.
kdeinit5: PID 933 terminated.
kdeinit5: Got EXEC_NEW '/usr/lib/qt/plugins/kf5/kio/tags.so' from launcher.
kdeinit5: preparing to launch '/usr/lib/qt/plugins/kf5/kio/tags.so'
kdeinit5: PID 628 terminated.
kdeinit5: PID 949 terminated.
klauncher: Exiting on signal 15
X connection to :0 broken (explicit kill or server shutdown).
